fmriprep/workflows/epi.py

```diff
@@ -161,47 +162,59 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         ])
 
     else:
-        LOGGER.info('Fieldmap estimation: type "%s" found', fmaps['type'])
-        # Import specific workflows here, so we don't brake everything with one
-        # unused workflow.
-        from fmriprep.workflows.fieldmap import init_fmap_estimator_wf, init_sdc_unwarp_wf
-        fmap_estimator_wf = init_fmap_estimator_wf(fmap_bids=fmaps,
-                                                   reportlets_dir=reportlets_dir,
+        # In case there are multiple fieldmaps prefer EPI
+        fmaps.sort(key=lambda fmap: {'epi': 0, 'fieldmap': 1, 'phasediff': 2}[fmap['type']])
+        fmap = fmaps[0]
+
+        LOGGER.info('Fieldmap estimation: type "%s" found', fmap['type'])
+
+        if fmap['type'] == 'epi':
+            epi_fmaps = [fmap['epi'] for fmap in fmaps if fmap['type']=='epi']
+            sdc_unwarp_wf = init_pepolar_unwarp_wf(fmaps=epi_fmaps,
```

Comment:
I would imagine `init_pepolar_unwar_wf` does not estimate the fieldmap only, it also corrects for distortion right?

I think that's perfect for now, but we should aim at the original design: one thing is the estimation and a different issue is applying that estimation to in fact correct for distortions.